web/frontend/src/components/do-command/index.svelte

```diff
@@ -1,87 +1,102 @@
 <script lang="ts">
-
 import { commonApi } from '@viamrobotics/sdk';
 import { notify } from '@viamrobotics/prime';
 import { resourceNameToString } from '@/lib/resource';
 import { doCommand } from '@/api/do-command';
 import Collapse from '@/lib/components/collapse.svelte';
 import { useRobotClient } from '@/hooks/robot-client';
+import { getClientByType } from './get-client-by-type';
+import { Button, Label, SearchableSelect } from '@viamrobotics/prime-core';
+import { isUnimplementedError } from './errors';
 
 export let resources: commonApi.ResourceName.AsObject[];
 
 const { robotClient } = useRobotClient();
 
-let selectedComponent = '';
-let input = '';
+$: selectedComponent = undefined as commonApi.ResourceName.AsObject | undefined;
```

Comment:
The `$:` here looks unnecessary

```suggestion
let selectedComponent: commonApi.ResourceName.AsObject | undefined;
```

Reply:
I was actually seeing some issues with things not behaving reactively without it, like the button staying disabled even when an option was selected. I ended up swapping some things around and using `core` components so I can go back and retest that.

Reply:
Yep, I think I fixed the reactivity issue I was seeing with the changes to how the handlers worker. Reverted it back to let.